plugins.yml

```diff
@@ -0,0 +1,11 @@
+# This is the central documentation for plugin settings.
+# Plugins not listed here are non-persistent and have priority 0
+
+# Plugin data, priority in range 0-9 inclusive, persistent is boolean
+PLUGINS:
+  epilepsy:
+    priority: 0
+    persistent: False
```

Comment:
"Persistent" indicates whether a particular plugin should be removed from the queue after executing for one cycle or remain on the queue. An example of the latter is an alarm, which should remain on the queue by default